backend/test/images/patch-csv.sh

```
@@ -0,0 +1,330 @@
+wait_for_dspo_controller_manager() {
+    local NAMESPACE_NAME="$1"
+
+    echo "Waiting for data-science-pipelines-operator-controller-manager pod to be ready..."
+    local dspo_pod_ready=false
+    local max_wait_time=300  # 5 minutes
+    local wait_interval=10   # Check every 10 seconds
+    local elapsed_time=0
+
+    while [[ $elapsed_time -lt $max_wait_time ]]; do
+        # Check if the DSPO controller manager pod exists and is running
+        local dspo_pod_status=$(oc get pods -n "$NAMESPACE_NAME" --field-selector=status.phase=Running --no-headers 2>/dev/null | grep "data-science-pipelines-operator-controller-manager" | wc -l)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)